app/views/users/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here your user with id</h1>
\ No newline at end of file
+<head>
+  <link rel="stylesheet" href="<%= asset_path('application.css') %>">
+</head>
+<div class= "photo-user">
+    <div class='user_img'>
+      <img src= <%= @user.photo %>  />
+    </div>
+<div class="user-card">
+<h1><%= @user.name  %></h1>
+<p>Number of posts:  <%= @user.posts_counter %> </p> 
+</div>
+</div>
+<div class="bio">
+<h2>Bio</h2>
+<p><span><%= @user.bio %></p>
+</div>
+<div>
+<% @user.recent_three_posts.each do |post| %>
+<div class="post">
+  <h2>Post :<%= link_to post.title, user_post_path(@user, post) %></h2>
+  <p><%= post.text %> </p>
+  Comments:  <%= post.comments_counter %> <br> Likes:  <%= post.likes_counter %>
```

Comment:
| Your User Show | Wireframe User Show |
| --- | --- |
| ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/6d756921-4678-4151-8d2b-55ca8e149590) | ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/3d23c9c9-6563-4840-af12-ebdc466777dc) |

- [ ] It would be nice if you make the image size and the content size similar as you can see in the wireframe to make your project follow the requirements :+1: but now your image size is a lot bigger than the expected size :fearful:
- [ ] It would be nice if you make the comments and likes counter at the right of the posts to match the wireframe :+1:
- [ ] It would be nice if you make your show all posts a button as you can see in the wireframe :+1:

Reply:
Thanks.